### PR TITLE
[New] LEMP Guides (1k)

### PR DESCRIPTION
**Pull request by Rajakavitha1**
- Added a new guide for CentOS 8
- Added a new guide for Debian 10
- Updated  the LEMP guide for Ubuntu 18.04

### CODE REVIEW

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit 81af9d6b87c6aa80fffd0c57ebaad35b33214455

https://deploy-preview-3207--nostalgic-ptolemy-b01ab8.netlify.com

**Comment by Rajakavitha1** [comment]
Here's the preview of the guides: https://deploy-preview-3207--nostalgic-ptolemy-b01ab8.netlify.com/web-servers/lemp/

File: docs/web-servers/lemp/how-to-install-the-lemp-stack-on-debian-10/index.md [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+
+    -  The site is served out of `/var/www/html/example.com/public_html` and its index page (`index.html`) is a simple `.html` file. **If your index page will use PHP like WordPress does, substitute `index.html` for `index.php`.**
+
+    -  `try_files` tells NGINX to verify that a requested file or directory [actually exists](https://nginx.org/en/docs/http/ngx_http_core_module.html#try_files) in the site's root filesystem before further processing the request. If it does not, a `404` is returned.
+
+    -  `location ~* \.php$` means that NGINX will apply this configuration to all .php files (file names are not case sensitive) in your site’s root directory, including any subdirectories containing PHP files.
+
+    -  The `*` in the `~* \.php$` location directive indicates that PHP file names are not case sensitive. This can be removed if you prefer to enforce letter case.
+
+    -  `fastcgi_pass` specifies the [UNIX socket](https://nginx.org/en/docs/http/ngx_http_fastcgi_module.html#fastcgi_pass) where PHP listens for incoming connections from other local processes
```
Comment (andystevensname) [inline docs/web-servers/lemp/how-to-install-the-lemp-stack-on-debian-10/index.md]: Period at the end of the sentence

File: docs/web-servers/lemp/how-to-install-the-lemp-stack-on-ubuntu-18-04/index.md [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+
+    -  The site is served out of `/var/www/html/example.com/public_html` and its index page (`index.html`) is a simple `.html` file. **If your index page will use PHP like WordPress does, substitute `index.html` for `index.php`.**
+
+    -  `try_files` tells NGINX to verify that a requested file or directory [actually exists](https://nginx.org/en/docs/http/ngx_http_core_module.html#try_files) in the site's root filesystem before further processing the request. If it does not, a `404` is returned.
+
+    -  `location ~* \.php$` means that NGINX will apply this configuration to all .php files (file names are not case sensitive) in your site’s root directory, including any subdirectories containing PHP files.
+
+    -  The `*` in the `~* \.php$` location directive indicates that PHP file names are not case sensitive. This can be removed if you prefer to enforce letter case.
+
+    -  `fastcgi_pass` specifies the [UNIX socket](https://nginx.org/en/docs/http/ngx_http_fastcgi_module.html#fastcgi_pass) where PHP listens for incoming connections from other local processes
```
Comment (andystevensname) [inline docs/web-servers/lemp/how-to-install-the-lemp-stack-on-ubuntu-18-04/index.md]: Period at the end of the sentence.

File: docs/web-servers/lemp/how-to-install-the-lemp-stack-on-ubuntu-18-04/index.md [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+
+5.  Enable the firewall to allow web traffic. This guide lists the commands to enable web traffic if you configured UFW on your server.
+
+      a. Check the ports that are enabled for `Nginx Full` Profile:
+
+        sudo ufw app info "Nginx Full"
+
+      Ports `80` and `443` should be listed as enabled for `Nginx Full` profile.
+
+      b. To allow incoming HTTP and HTTPS traffic for `Nginx Full` profile:
```
Comment (andystevensname) [inline docs/web-servers/lemp/how-to-install-the-lemp-stack-on-ubuntu-18-04/index.md]: As previously mentioned, why the a. and b. ? Sentence should also read "if these ports are not allowed, enable them..." or something similar.

File: docs/web-servers/lemp/how-to-install-the-lemp-stack-on-debian-10/index.md [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+
+5.  Enable the firewall to allow web traffic. This guide lists the commands to enable web traffic if you configured UFW on your server.
+
+      a. Check the ports that are enabled for `Nginx Full` Profile:
+
+        sudo ufw app info "Nginx Full"
+
+      Ports `80` and `443` should be listed as enabled for `Nginx Full` profile.
+
+      b. To allow incoming HTTP and HTTPS traffic for `Nginx Full` profile:
```
Comment (andystevensname) [inline docs/web-servers/lemp/how-to-install-the-lemp-stack-on-debian-10/index.md]: As previously mentioned, why a. and b.? Sentence should also read "if the ports are not enabled" or something similar.

Comment (hzoppetti) [reply docs/web-servers/lemp/how-to-install-the-lemp-stack-on-debian-10/index.md]: restructured this to be a new section

File: docs/web-servers/lemp/how-to-install-the-lemp-stack-on-ubuntu-18-04/index.md (hunk 3 above)
Comment (hzoppetti) [reply docs/web-servers/lemp/how-to-install-the-lemp-stack-on-ubuntu-18-04/index.md]: restructured this to be a new section

File: docs/web-servers/lemp/how-to-install-the-lemp-stack-on-ubuntu-18-04/index.md (hunk 2 above)
Comment (hzoppetti) [reply docs/web-servers/lemp/how-to-install-the-lemp-stack-on-ubuntu-18-04/index.md]: done

File: docs/web-servers/lemp/how-to-install-the-lemp-stack-on-debian-10/index.md (hunk 1 above)
Comment (hzoppetti) [reply docs/web-servers/lemp/how-to-install-the-lemp-stack-on-debian-10/index.md]: done

File: docs/web-servers/lemp/how-to-install-the-lemp-stack-on-centos-8/index.md [hunk 5]
```diff
@@ -0,0 +1,261 @@
...
+2. Assign the ownership of the directory to `$USER`:
+
+        sudo chown -R $USER:$USER /var/www/html/example.com/public_html
+
+1.  Use SELinux’s `chcon` command to change the file security context for web content:
+
+        sudo chcon -t httpd_sys_content_t /var/www/html/example.com -R
+        sudo chcon -t httpd_sys_rw_content_t /var/www/html/example.com -R
+
+3.  Update the document root location of the web content in `/etc/nginx/nginx.conf` file.
```
[the remark this answers is not in the document]
Comment (hzoppetti) [reply docs/web-servers/lemp/how-to-install-the-lemp-stack-on-centos-8/index.md]: added

File: docs/web-servers/lemp/how-to-install-a-lemp-server-on-ubuntu-18-04/index.md [hunk 6]
```diff
@@ -151,6 +150,19 @@ server {
...
 
+5.  Enable the firewall to allow web traffic. This guide lists the commands to enable web traffic if you configured UFW on your server.
+
+      a. Check the ports that are enabled for `Nginx Full` Profile:
+
+          sudo ufw app info "Nginx Full"
+
+      Ports `80` and `443` should be listed as enabled for `Nginx Full` profile.
+
+      b. To allow incoming HTTP and HTTPS traffic for `Nginx Full` profile:
```
[the remark this answers is not in the document]
Comment (hzoppetti) [reply docs/web-servers/lemp/how-to-install-a-lemp-server-on-ubuntu-18-04/index.md]: removed this guide

File: docs/web-servers/lemp/how-to-install-a-lemp-server-on-ubuntu-18-04/index.md [hunk 7]
```diff
@@ -151,6 +150,19 @@ server {
 
         sudo ln -s /etc/nginx/sites-available/example.com.conf /etc/nginx/sites-enabled/
 
+5.  Enable the firewall to allow web traffic. This guide lists the commands to enable web traffic if you configured UFW on your server.
+
+      a. Check the ports that are enabled for `Nginx Full` Profile:
```
[the remark this answers is not in the document]
Comment (hzoppetti) [reply docs/web-servers/lemp/how-to-install-a-lemp-server-on-ubuntu-18-04/index.md]: removed this guide

File: docs/web-servers/lemp/how-to-install-a-lemp-server-on-ubuntu-18-04/index.md [hunk 8]
```diff
@@ -102,11 +102,10 @@ quit
 
         sudo mkdir -p /var/www/html/example.com/public_html
 
-2.  Delete the default site configuration provided with the package as an example:
+2.  Create a copy of the default configuration file for your site:
```
[the remark this answers is not in the document]
Comment (hzoppetti) [reply docs/web-servers/lemp/how-to-install-a-lemp-server-on-ubuntu-18-04/index.md]: removed this guide

**Pull request merged**